src/Dfc.CourseDirectory.Web/Views/BulkUpload/Index.cshtml

```diff
@@ -75,27 +65,9 @@
 
         <form method="post" id="bulkUploadForm" enctype="multipart/form-data" asp-controller="BulkUpload" asp-action="Index">
 
-            <p class="govuk-body">You can upload a CSV file to update your courses, or you can:</p>
-            <ul class="govuk-list govuk-list--bullet">
-                <li><a asp-controller="RegulatedQualification" asp-action="Index" class="govuk-link">add a new course</a></li>
-                <li><a asp-controller="ProviderCourses" asp-action="Index" class="govuk-link">edit or delete each course individually</a></li>
-            </ul>
-
-            <h2 class="govuk-heading-m">Creating your bulk upload file</h2>
-
-            <p class="govuk-body">Uploading and publishing a bulk upload file will replace all of your published courses. To create the file you can:</p>
-
-            <ul class="govuk-list govuk-list--bullet">
-                <li><a asp-action="GetBulkUploadTemplateFile" asp-controller="BlobStorage" class="govuk-link">download a blank template</a> and add your course information</li>
-                <li>download and edit a file containing your <a asp-action="GetCurrentCoursesTemplateFile" asp-controller="BlobStorage" class="govuk-link">published course information</a></li>
-            </ul>
-
-            <h2 class="govuk-heading-m">If you have 800 rows or fewer in your bulk upload file</h2>
-
             <div id="groupServerSideError" class="govuk-form-group">
-                <label class="govuk-label" for="bulkUploadFile">Upload a file</label>
                 <span id="serverSideError" class="govuk-error-message" style="display: none">There is an error with the file. See above summary</span>
-                <input class="govuk-file-upload" id="bulkUploadFile" name="bulkUploadFile" type="file"
+                <input class="govuk-file-upload" id="bulkUploadFile" name="bulkUploadFile" type="file" accept=".csv"
```

Comment:
This seemed like low hanging fruit as an improvement considering we only accept csv files... thoughts?

Reply:
Yep seems totally reasonable